rgw/v2/tests/s3_swift/multisite_configs/test_check_sharding_enabled_brownfield.yaml

```diff
@@ -0,0 +1,3 @@
+config:
+  cluster_type: brownfield
+  enable_sharding: true
```

Comment:
i also think that you need to add `enable_sharding` in the config object and default to `False`